src/JSONLD/JSONLD.jl

```diff
@@ -0,0 +1,86 @@
+abstract type JSONLD <: DataRepo
+end
+
+export JSONLD_Web, JSONLD_DOI
```

Comment:
All exports go in src/DataDepsGenerators.jl

Reply:
This is how I used to do for abstracted types like `DataOnev2`.

Reply:
well it isn't a big deal and it is trivial to change later. So I guess either is good.